dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/RelativePeriods.java

```diff
@@ -1316,6 +1337,19 @@ public RelativePeriods setLast5Years( boolean last5Years )
         return this;
     }
```

Comment:
I think we need to also update `getRelativePeriodEnums` and `setRelativePeriodsFromEnums`.

Reply:
> We also need to update `RelativePeriods.hbm.xml` to add this new relative period for persistence. We store relative periods as part of visualizations and maps.

Done

Reply:
Enums as well

@larshelge, how should be  the hbm change distributed to the database? Now it CAN'T BE MERGED, but you know it for sure